tests/wicked/aggregate/sut/t10_bonding_ab_ping_2_ips.pm

```diff
@@ -0,0 +1,36 @@
+# SUSE's openQA tests
+#
+# Copyright © 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Bonding, Active-Backup Ping 2 IPs
+# Maintainer: Anton Smorodskyi <asmorodskyi@suse.com>
+#             Jose Lausuch <jalausuch@suse.com>
+#             Clemens Famulla-Conrad <cfamullaconrad@suse.de>
+
+use Mojo::Base 'wickedbase';
+use testapi;
+
+
+sub run {
+    my ($self, $ctx) = @_;
+    record_info('INFO', 'Bonding, Active-Backup Ping 2 IPs');
+    $self->setup_bond('alb', $ctx->iface(), $ctx->iface2());
+    $self->validate_interfaces('bond0', $ctx->iface(), $ctx->iface2());
+    my $active_link1 = $self->get_bond_active_link('bond0');
+    sleep 5;    # Wait some seconds until bond process fails over to the other interface
+    assert_script_run("ip link set dev $active_link1 down");
```

Comment:
have you seen Marius answer on my question ? 
https://w3.suse.de/~mt/tools/ifbind.sh  and `ifbind.sh unbind eth1`
https://gitlab.suse.de/wicked-maintainers/wicked/issues/21
In any case I would go for replacing this line with one-liner function ( yes I am saying this :) ) . So we would have generic way of shutting down interfaces which we could improve later

Reply:
done